docs/source/usage.rst

```diff
@@ -118,6 +118,9 @@ Use the docker image as pre-commit hook by adding the docker run command to ``.g
     docker run -t --rm -v "$PWD:/git" godaddy/tartufo --pre-commit
     EOF
 
+
+.. _examplecleanup:
```

Comment:
Similar to the `.. _usage:` on the index, this shouldn't be here; it turns this into an anchor, so that the link below simply points here.